client/agent-opentracing/src/main/resources/apmrules/java/apm-java-net.btm

```diff
@@ -9,7 +9,8 @@ AT ENTRY
 IF !$0.connected && includePath($0.getURL().getPath()) && !callerMatches("HttpURLConnection.*",true)
 DO
   startSpanWithParent(getTracer().buildSpan($0.getRequestMethod())
-  		.withTag("http.uri", $0.getURL().getPath())
+  		.withTag("span.kind", "client")
+  		.withTag("http.url", $0.getURL().toString())
```

Comment:
Yes 'http.uri' was an alternative in zipkin, but OT has standardised on `http.url`.